Game/Scenes/TestScene.json

```diff
@@ -32,7 +32,8 @@
 				"y" : 1.0,
```

Comment:
Every UID in this file is zero, which isn't a unique identifier, which might be unintended behaviour.

Reply:
They are all given a UID = 0, since they are only given a UID when you create the entities. Loading them from a .json file does not give each entity a new UID.